packages/cfd/src/Helpers/compare-accounts-config.ts

```diff
@@ -14,92 +14,95 @@ const getHighlightedIconLabel = (
 ): TInstrumentsIcon[] => {
     const market_type = getMarketType(trading_platforms);
     const market_type_shortcode = market_type.concat('_', trading_platforms.shortcode);
-    // Forex for these: MT5 Financial Vanuatu, MT5 Financial Labuan
     const forex_label =
-        market_type_shortcode === MARKET_TYPE_SHORTCODE.FINANCIAL_LABUAN ||
-        market_type_shortcode === MARKET_TYPE_SHORTCODE.FINANCIAL_VANUATU ||
-        is_demo ||
-        trading_platforms.platform === CFD_PLATFORMS.DXTRADE ||
-        trading_platforms.platform === CFD_PLATFORMS.CTRADER ||
-        selected_region === REGION.EU ||
-        (trading_platforms.platform === CFD_PLATFORMS.MT5 && market_type_shortcode === 'all_svg')
+        selected_region === REGION.EU
             ? localize('Forex')
+            : market_type_shortcode === MARKET_TYPE_SHORTCODE.FINANCIAL_LABUAN
+            ? localize('Forex: standard/exotic')
+            : (trading_platforms.platform === CFD_PLATFORMS.MT5 && market_type_shortcode === 'all_svg') ||
+              trading_platforms.platform === CFD_PLATFORMS.CTRADER
+            ? localize('Forex: major/minor')
```

Comment:
Please avoid using multiple ternaries. Eslint will complain about it.
Can use instead of another variable for falsy value.

Reply:
```
const getForexType = (selected_region, market_type_shortcode, trading_platforms) => {
    if (selected_region === REGION.EU) {
        return localize('Forex');
    } else if (market_type_shortcode === MARKET_TYPE_SHORTCODE.FINANCIAL_LABUAN) {
        return localize('Forex: standard/exotic');
    } else if (
        (trading_platforms.platform === CFD_PLATFORMS.MT5 && market_type_shortcode === 'all_svg') ||
        trading_platforms.platform === CFD_PLATFORMS.CTRADER
    ) {
        return localize('Forex: major/minor');
    } else {
        return localize('Forex: standard/micro');
    }
};
```

// Usage
```
const forexType = getForexType(selected_region, market_type_shortcode, trading_platforms);
```